quattor/types/metadata.pan

```diff
@@ -4,6 +4,10 @@ type structure_branch = {
     "author" ? string
     "name" : string
     "type" : string
+    @{a unique id, e.g. hash of last git commit}
+    "id" ? string
```

Comment:
i thought about naming this version, but version suggests sometyhing ordered, which a commit hash is not. i can add another optional version field (anf maybe retrict it to `\d+.\d+\.\d+`)